client/src/components/DisplayTable.js

```diff
@@ -71,50 +79,54 @@ export class DisplayTable extends React.Component {
     } = this.state;
 
     const clean_search_string = (search_string) => _.chain(search_string).deburr().toLower().trim().value();
-    const sorted_filtered_data = _.chain(rows)
-      .filter(
-        ({search_values}) => _.chain(search_values)
-          .map( (search_value, column_key) => (
-            _.isEmpty(searches[column_key]) ||
+    const sorted_filtered_data = _.chain(data)
+      .filter( (row) => _.chain(row)
+        .map( (column_value, column_key) => (
+          _.isEmpty(searches[column_key]) ||
             _.includes(
-              clean_search_string(search_value),
+              clean_search_string(column_value),
               clean_search_string(searches[column_key])
             )
-          ) )
-          .every()
-          .value()
+        ) )
+        .every()
+        .value()
       )
-      .sortBy( ({sort_values}) => _.isNumber(sort_values[sort_by]) ? sort_values[sort_by] : Number.NEGATIVE_INFINITY )
+      .sortBy( row => row[sort_by] && (
+        column_configs[sort_by].sort_func ?
```

Comment:
Lodash currently doesn't support comparator natively: https://github.com/lodash/lodash/pull/3764
I think I can add mixin to make it work though

Reply:
Always forget that. The JS built in array sort takes a comparator, could do
```
  // ...
  .thru( (unsorted_array) => unsorted_array.sort(
    (a, b) => {
      // whatever
    }
  ) )
  // ...
```

Reply:
A mixin would be handy though!